lib/adagrams.rb

```diff
@@ -0,0 +1,80 @@
+require 'pry'
+
+
+def draw_letters
+  letters_in_hand = ARRAY_OF_LETTERS.shuffle.slice(1..10)
+  return letters_in_hand
+end
+
+ARRAY_OF_LETTERS = ["A", "A", "A", "A", "A", "A", "A", "A", "A", "B", "B", "C", "C", "D", "D", "D", "D", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "F", "F", "G", "G", "G", "H", "H", "I", "I", "I", "I", "I", "I", "I", "I", "I", "J", "K", "L", "L", "L", "L", "M", "M", "N", "N", "N", "N", "N", "N", "O", "O", "O", "O", "O", "O", "O", "O", "P", "P", "Q", "R", "R", "R", "R", "R", "R", "S", "S", "S", "S", "T", "T", "T", "T", "T", "T", "U", "U", "U", "U", "V", "V", "W", "W", "X", "Y", "Y", "Z"]
+
+def uses_available_letters? (input, letters_in_hand)
+  input = input.upcase.chars
+  dup_letters_in_hand = letters_in_hand.clone
+  
+  
+  input.each do |chars|
+    if dup_letters_in_hand.include?(chars)
+      dup_letters_in_hand.delete(chars)
+    else 
+      return false
+    end
+  end
+  
+  check = 0
```

Comment:
This check seems redundant. The first loop will return false if a letter in the word is not in the hand. If that return doesn't happen, you can simply return true after that loop is finished.